src/rpc/mining.cpp

```diff
@@ -380,6 +382,7 @@ static UniValue getblocktemplate(const JSONRPCRequest& request)
             "       ]\n"
             "     }\n"
             "\n"
+            "2. algo    (string, optional) The mining algorithm to use for this pow hash, 'progpow', 'sha256d', 'randomx'\n"
```

Comment:
Please add note if not set randomx is the default.

Reply:
Will do

Reply:
Done